python/cloudtik/templates/aliyun/latest_cascade_lake/large-highmem.yaml

```diff
@@ -0,0 +1,11 @@
+from: aliyun/latest_cascade_lake/large
```

Comment:
"latest" is dynamic reference to the latest version but doesn't mention directly so that when the version is changed, we don't change.

For now, ice lake is the latest. and in the half SPR is the latest. So the name is not change but we will update the instance type in the config file.

If we do need call out specifically cascade lake, we can use the following way:
We change "latest_cascade_lake" to "cascade_lake"
We change "latest_ice_lake" to "latest"

When SPR become the latest, we change the old latest to "ice_lake" and the new "latest" is actually SPR.

Reply:
@jerrychenhf Hi haifeng, I will refactor the name as your comments. Thank.